### PR TITLE
Update obcs.md

### PR DESCRIPTION
**Pull request by zhengj2007**
Updated contact person email and license and added usage

### CODE REVIEW

File: ontology/obcs.md
Comment (cthoyt) [inline ontology/obcs.md:31]: I can't comment on whether these usages are adequate or not

Comment (zhengj2007) [reply ontology/obcs.md:31]: Thanks, @cthoyt. Who do you think I can request to review?

**Comment by yongqunh** [comment]
I checked and approve the updates.

**Review by yongqunh — APPROVED** [review]
I approve the change.

**Review by deepakunni3 — CHANGES_REQUESTED** [review]

File: ontology/obcs.md
Comment (zhengj2007) [reply ontology/obcs.md:28]: The terms are used in the backend for organizing the studies. It is not viewable on the website.

Comment (deepakunni3) [reply ontology/obcs.md:28]: Thank you for clarifying

File: ontology/obcs.md
Comment (deepakunni3) [reply ontology/obcs.md]: Yes, will take a look at the failing test 👍

Comment (zhengj2007) [reply ontology/obcs.md]: Thanks @deepakunni3

File: ontology/obcs.md
Comment (matentzn) [reply ontology/obcs.md:28]: Unless the website provides some form of real evidence that the ontology used it cannot be used as evidence for `usage`. This link does not return anything for me!

File: ontology/obcs.md
Comment (matentzn) [reply ontology/obcs.md]: Papers must be using either pmid or doi URLs.

**Review by matentzn — CHANGES_REQUESTED** [review]
Please only provide evidence for usages that can be verified. Else anyone can add whatever they want in the usage section!

File: ontology/obcs.md
Comment (zhengj2007) [reply ontology/obcs.md]: changed the paper url to doi one.

File: ontology/obcs.md
Comment (zhengj2007) [reply ontology/obcs.md]: Removed this usage

File: ontology/obcs.md
Comment (matentzn) [inline ontology/obcs.md:29]: @zhengj2007 I cant say anything against this because it is clearly in the OBO principles. But since you are OBO Ops I just want to mention that this kind of "usage evidence" is not really very robust. This just proves that OBCS has been used in a study at some point, not that it is "actively used". I will leave it to @deepakunni3 to use his authority to make a call here, but in my view this is:

1. Formally you have every right to get this approved, as it is stated in our principles
2. If we want to really use the `usages` field correctly, providing papers as evidence is not in the spirit of the inventor because everyone can write a word document explain how they used the ontology for NLP, upload it to Zenodo and record it here - which makes this field meaningless.

Comment (zhengj2007) [reply ontology/obcs.md:29]: @matentzn I know OBCS is used in [VEUPATH](https://veupathdb.org/). But I cannot find the link to the term since many ontology terms used in the  [VEUPATH](https://veupathdb.org/) annotations did not show the ontology terms directly. I can find some OBO ontologies import OBCS, such as [CIDO](https://ontobee.org/ontostat/catalog/CIDO?prefix=OBCS&iri=http://www.w3.org/2002/07/owl%23Class) and [EUPATH](https://ontobee.org/ontostat/catalog/EUPATH?prefix=OBCS&iri=http://www.w3.org/2002/07/owl%23Class)? Can I put the import as a usage?

I tried to add the usage section because it is a required field for passing the dashboard checking. Actually, I'd prefer to change it as warn rather than error for the first stage dashboard QC.

Comment (matentzn) [reply ontology/obcs.md:29]: Haha, you are talking to the wrong person. Sorry; I would like to be more strict here, as `usage` is a really important and hard-to-assess metric; imports are two easy again IMO to justify usage. You wont hear something better from me; I hear you want OBCS to pass the dashboard, and I wont stand in the way of that, as you are adhering to the verbatim of our rules - just saying I am not a fan of the way the rules are formulated.

If we set usage to "WARN", we basically decided that we don't care about this principle.. No one will ever make an effort to fill it in.

A better way IMO would be to find a way to prove that OBCS is used VEUPATH. Working with the VEUPATH people to update the pages and expose the OBCS IDs; even a data source documentation showing the use of OBCS would be ok.

Comment (zhengj2007) [reply ontology/obcs.md:29]: @matentzn I tried to find the OBCS terms in the VEUPATH. However, I left the VEUPATH team and cannot access the backend codes now. It is too hard to try to get the OBCS terms from over thousands of studies. The team cut whole ontology team. It's easy hard to ask them to document the ontologies used in the project. 

I don't say usage is not important. However, the ontology is hard to show strong usage evidences at beginning of development. But the developers have needs when they developed an ontology. What I mean for the first dashboard QC stage, I'd prefer to set it as WARN. Otherwise, most of new ontologies such as OccO will show error.

Comment (matentzn) [reply ontology/obcs.md:29]: I hear you! I just personally dont agree (neither with watering down usages to WARN, nor with allowing usage documentation without a life system and proof), but, the law is on your side, so you don't need my agreement! :)

Comment (matentzn) [reply ontology/obcs.md]: @deepakunni3 This debate has been finished in #2300 (see associated PRs), and I don't think we should rip this open again right now. From your end, you can only assess if @zhengj2007 adheres to this definition (which I think she does, but didn't dig deep).

**Review by deepakunni3 — APPROVED** [review]

**Comment by deepakunni3** [comment]
@matentzn Would need your approval so that this PR can be merged :)

**Review by matentzn — APPROVED** [review]

**Pull request merged**
